CONTRIBUTING.md

```diff
@@ -52,7 +52,7 @@ SMS_TEMPLATE_ID "valid sms_template_id"
 LETTER_TEMPLATE_ID "valid letter_template_id"
 EMAIL_REPLY_TO_ID "valid email reply to id"
 SMS_SENDER_ID "valid sms_sender_id - to test sending to a receiving number, so needs to be a valid number"
-API_SENDING_KEY "API_whitelist_key for sending an SMS to a receiving number"
+API_SENDING_KEY "API_whitelist_key for sending a SMS to a receiving number"
```

Comment:
The correct article to use here depends on how the acronym is pronounced, not whether the first letter is a vowel. Because SMS is pronounced ‘ess-em-ess’ it starts with a vowel sound, ie it’s ‘an SMS’.

Similarly it’s _a_ MOMA exhibit but _an_ MRI scan; _an_ FBI agent but _a_ FEMA agent.

Reply:
ok, never knew that